### DISCUSSION TITLE
Images png in rmd not load in webpag of netlify.app in Hugo clarity theme

### DISCUSSION THREAD

**Discussion by ghost** [chipzoller/hugo-clarity#196] [General]
**I have checked all the prerequisites below and I'm yet experiencing a problem**

- [ ] Read the README.md
- [ ] Have the extended version of Hugo installed
- [ ] Used the exampleSite's config.toml as a reference

**Good afternoon:
I have uploaded the clarity theme to my website with netlify and I have noticed that the images that load in the rmarkdown document do not load in the html and therefore in the web page. I would like to know what I have done wrong about it in order to improve**

```yaml
title: “Volvamos a sentirnos peruanos”
author: “Marco Parinango”
date: ‘2021-06-16’
slug: peruanos
categories: []
tags: []
description: Article description.
featured: yes
toc: no
featureImage: images/juntos.png
thumbnail: /images/path/thumbnail.png
shareImage: images/juntos.png
codeMaxLines: 10
codeLineNumbers: no
figurePositionShow: yes
fontsize: 11pt
lang: en
insertando imagen
```

C:_gen.png

library(knitr)
library(png)
img1_path <- "C:/Users/gigar/Documents/GitHub/perspectiva/resources/_gen/images/juntos.png" 
img1 <- readPNG(img1_path, native = TRUE, info = TRUE)
attr(img1, "info")

![{0A71672C-432A-457D-AECC-5AD0B678E34D}](https://user-images.githubusercontent.com/13245618/122322774-a07b3e80-ceeb-11eb-9a9f-08dcddbd755a.png) # from rmd

![{64726FD5-F1F5-442B-ACB9-432464F48857}](https://user-images.githubusercontent.com/13245618/122323583-eedd0d00-ceec-11eb-907e-b910bd5bdb13.png) #from html file

[https://perspectiva.netlify.app/post/peruanos/](url)

**Desktop (please complete the following information):**
 - OS:  Windows 10]
 - Browser [Google Chrome está actualizado
Versión 91.0.4472.106 (Build oficial) (64 bits)]

blogdown::check_site()
― Running a series of automated checks for your blogdown website project...
------------------------------------------------------------------------------------------------
○ A successful check looks like this.
● [TODO] A check that needs your attention looks like this.
| Let's check out your blogdown site!
------------------------------------------------------------------------------------------------
― Checking config.toml
| Checking "baseURL" setting for Hugo...
○ Found baseURL = "https://perspectiva.netlify.app/"; nothing to do here!
| Checking "ignoreFiles" setting for Hugo...
● [TODO] Set "ignoreFiles" to ["\\.Rmd$", "\\.Rmarkdown$", "_cache$", "\\.knit\\.md$", "\\.utf8\\.md$"]
| Checking setting for Hugo's Markdown renderer...
○ All set! Found the "unsafe" setting for goldmark.
― Check complete: config.toml

― Checking .gitignore
| Checking for items to remove...
○ Nothing to see here - found no items to remove.
| Checking for items to change...
○ Nothing to see here - found no items to change.
| Checking for items you can safely ignore...
● [TODO] You can safely add to .gitignore: .DS_Store, Thumbs.db
| Checking for items to ignore if you build the site on Netlify...
○ Found! You have safely ignored: /public/, /resources/
| Checking for files required by blogdown but not committed...
○ Great! Did not find such files.
― Check complete: .gitignore

― Checking Hugo
| Checking Hugo version...
○ Found Hugo. You are using Hugo 0.83.1.
| Checking .Rprofile for Hugo version used by blogdown...
○ blogdown is using Hugo 0.83.1 to build site locally.
― Check complete: Hugo

― Checking netlify.toml...
○ Found HUGO_VERSION = 0.83.1 in [build] context of netlify.toml.
| Checking that Netlify & local Hugo versions match...
○ It's a match! Blogdown and Netlify are using the same Hugo version (0.83.1).
| Checking that Netlify & local Hugo publish directories match...
○ Good to go - blogdown and Netlify are using the same publish directory: public
― Check complete: netlify.toml

― Checking content files
| Checking for validity of YAML metadata in posts...
○ All YAML metadata appears to be syntactically valid.
| Checking for previewed content that will not be published...
○ Found 0 files with future publish dates.
○ Found 0 files marked as drafts.
| Checking your R Markdown content...
○ All R Markdown files have been knitted.
○ All R Markdown output files are up to date with their source files.
| Checking for .html/.md files to clean up...
○ Found 0 duplicate .html output files.
○ Found 0 incompatible .html files to clean up.
| Checking for the unnecessary 'content/' directory in theme...
○ Great! Your theme does not contain the content/ directory.
― Check complete: Content

![{64726FD5-F1F5-442B-ACB9-432464F48857}](https://user-images.githubusercontent.com/13245618/122323925-82aed900-ceed-11eb-8143-d73bd0abd06f.png)

xfun::session_info()
R version 4.0.5 (2021-03-31)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042), RStudio 1.4.1106

Locale:
  LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252   
  LC_MONETARY=English_United States.1252 LC_NUMERIC=C                          
  LC_TIME=English_United States.1252    

Package version:
  base64enc_0.1.3   blogdown_1.3.2    bookdown_0.22     compiler_4.0.5   
  digest_0.6.27     evaluate_0.14     glue_1.4.2        graphics_4.0.5   
  grDevices_4.0.5   highr_0.9         htmltools_0.5.1.1 httpuv_1.6.1     
  jsonlite_1.7.2    knitr_1.33        later_1.2.0       magrittr_2.0.1   
  markdown_1.1      methods_4.0.5     mime_0.10         processx_3.5.2   
  promises_1.2.0.1  ps_1.6.0          R6_2.5.0          Rcpp_1.0.6       
  rlang_0.4.11      rmarkdown_2.8     rstudioapi_0.13   servr_0.22       
  stats_4.0.5       stringi_1.6.2     stringr_1.4.0     tinytex_0.32     
  tools_4.0.5       utils_4.0.5       xfun_0.23         yaml_2.2.1

**Comment by onweru** [comment]
@carlosgino, please link your repo here, it's easier to look at it instead.
